### PR TITLE
feat: add aborted view for Webpay-plus

### PR DESCRIPTION
**Pull request by DiegoHBustamante**
This versión includes the following changes:

- Add aborted view to cancel purchase in webpay-plus.
- Add condition in WebpayPlusController to redirect to the aborted view when the cancel purchase action is performed.

These changes were made because currently the example project in the Webpay-plus section was redirecting to the error.jsp view when a purchase cancellation was performed.

> Case without modifications
Captura de Pantalla 2024-06-17 a la(s) 18 37 33

> Case with modifications

### CODE REVIEW

File: src/main/webapp/WEB-INF/jsp/webpay_plus/aborted.jsp [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+<style>
+	.row {
+		padding-bottom: 20px;
+	}
+</style>
+
+<div class="row">
+	<div class="col">
+		 <h2>Compra cancelada por el usuario</h2>
+         <p>Luego de que se anula la compra en el formulario de pago recibir�s un POST con lo siguiente:</p>
```
Comment (tribiec) [inline src/main/webapp/WEB-INF/jsp/webpay_plus/aborted.jsp]: Are we sure that it's a POST and not a GET request?

**Review by tribiec — APPROVED** [review]

File: src/main/webapp/WEB-INF/jsp/webpay_plus/aborted.jsp (hunk 1 above)
Comment (DiegoHBustamante) [reply src/main/webapp/WEB-INF/jsp/webpay_plus/aborted.jsp]: The corresponding changes will be made

File: src/main/webapp/WEB-INF/jsp/webpay_plus/aborted.jsp
Comment (mastudillot) [inline src/main/webapp/WEB-INF/jsp/webpay_plus/aborted.jsp:1]: add empty line

**Review by mastudillot — CHANGES_REQUESTED** [review]
check my comment

File: src/main/webapp/WEB-INF/jsp/webpay_plus/aborted.jsp
Comment (DiegoHBustamante) [reply src/main/webapp/WEB-INF/jsp/webpay_plus/aborted.jsp:1]: add an empty line between <h2> y <p>?

Comment (DiegoHBustamante) [reply src/main/webapp/WEB-INF/jsp/webpay_plus/aborted.jsp:1]: added the empty line

**Review by mastudillot — APPROVED** [review]
LGTM

**Pull request merged**